cmd/web/middleware.go

```diff
@@ -25,3 +25,14 @@ func (app *application) RecoveryMiddleware(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r)
 	})
 }
+
+func secureHeaders(next http.Handler) http.Handler {
```

Comment:
`app` struct is used as a dependencies provider in handlers level. Because `secureHeaders` does not need any dependencies from `app` it's declared as standalone function, but for consistency it's good idea to make it method of `app`.

Reply:
Resolved with [227f2b9](https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-fdemchenko/pull/4/commits/227f2b91220cc70c3065a40d7bdb909270104116)